contracts/multisig/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
The top level changelog can link to each contracts changelog maybe. Do you wanna move ampd Changelog into it's folder too?

Reply:
that's a good idea, but ill do that as a separate PR